### PR TITLE
Common Accessibility Remedial document 2nd version

### PR DESCRIPTION
**Pull request by sangitasmane**

### CODE REVIEW

File: accessibility/common-remediations.md [hunk 1]
```diff
@@ -17,6 +18,23 @@ This page describes some common accessibility issues, and gives examples and sug
...
+   - [Same title used on multiple pages](#same-title-used-on-multiple-pages)
+
+- [Headings](#headings)
+   - [Headings used for styling](#headings-used-for-styling)
+   - [Implied headings](#implied-headings)
+   - [Multiple H1s per page](#multiple-H1s-per-page)
+   - [Heading sequence is not logical](#heading-sequence-is-not-logical)
+
+- [HTML Attributes](#html-attributes)
+   - [Reliance on title attributes](#reliance-on-title-attributes)
```
Comment (joeyciechanowicz) [inline accessibility/common-remediations.md]:
```suggestion
   - [Reliance on title attributes for accessibility](#Reliance on title attributes for accessibility)
```

File: accessibility/common-remediations.md [hunk 2]
```diff
@@ -17,6 +18,23 @@ This page describes some common accessibility issues, and gives examples and sug
    - [Redundant alt attributes](#redundant-alt-attributes)
    - [Brand logos](#brand-logos)
 
+- [Page titles](#page-titles)
+   - [Title doesn’t describe the page](#title-doesn’t-describe-the-page)
```
Comment (joeyciechanowicz) [inline accessibility/common-remediations.md]:
```suggestion
   - [Title doesn’t describe the page](#title-doesnt-describe-the-page)
```

File: accessibility/common-remediations.md [hunk 3]
```diff
@@ -219,3 +210,185 @@ We don't do this:
...
+
+## Headings  
+
+### Headings used for styling
+
+Do not use semantic heading tags to control the appearance of text. Use CSS for styling.
+
+We do this:
+```html
+<h2 class=”level-4-styling”>This content needs a level 2 heading but I want it to look like a level 4 heading</h2>
```
Comment (joeyciechanowicz) [inline accessibility/common-remediations.md]: I've generally seen us using classes named `h1, h2, ...` etc
```suggestion
<h2 class=”h4”>This content needs a level 2 heading but I want it to look like a level 4 heading</h2>
```

**Comment by sangitasmane** [comment]
Thanks @joeyciechanowicz will keep check on this next time!

**Pull request merged**